apps/angular/src/app/app.routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Routes } from '@angular/router';
+
+import { AnimeCatalogComponent } from './features/anime-catalog/anime-catalog.component';
+
+/** Routes object. */
+export const appRoutes: Routes = [
+	{
+		path: '',
+		component: AnimeCatalogComponent,
```

Comment:
```suggestion
		loadComponent: ...,
```
Since you are using standalone components, you can use lazy loading for them
https://ultimatecourses.com/blog/lazy-load-standalone-components-via-load-component